test/lit/passes/local-subtyping.wast

```diff
@@ -568,4 +571,50 @@
       (local.get $x)
     )
   )
+
+  ;; CHECK:      (func $try_table-catch-result (type $0)
+  ;; CHECK-NEXT:  (drop
+  ;; CHECK-NEXT:   (block $catch (result anyref)
+  ;; CHECK-NEXT:    (try_table (catch $e-anyref $catch)
+  ;; CHECK-NEXT:     (nop)
+  ;; CHECK-NEXT:    )
+  ;; CHECK-NEXT:    (ref.null none)
+  ;; CHECK-NEXT:   )
+  ;; CHECK-NEXT:  )
+  ;; CHECK-NEXT: )
+  (func $try_table-catch-result
+    (drop
+      ;; Must not be refined to nullref.
+      ;; An anyref comes from the tag's payload because it is not a catch_all.
+      (block $catch (result anyref)
```

Comment:
Does this mean that this `(result anyref)` should not be refined to something more refined than `anyref`? Does `LocalSubtyping` refine `block`'s return types too? (I'm not familiar with the pass)

And I think `it is not a catch_all` in the comment may not be necessary.

Reply:
> Does this mean that this `(result anyref)` should not be refined to something more refined than `anyref`?

Yes, that's it. I specifically added "(result nullref)" in the comment.

> Does `LocalSubtyping` refine `block`'s return types too? (I'm not familiar with the pass)

It does through `ReFinalize`. It's a bit weird to me that `ReFinalize::visitNode` and `Node::finalize` are separate but seem to do very similar things. For some passes, `finalize` is what matters; for others (like `LocalSubtyping`), it's `ReFinalize`.